src/eko/scale_variations/exponentiated.py

```diff
@@ -43,3 +43,63 @@ def gamma_variation(gamma, order, nf, L):
     if order[0] >= 2:
         gamma[1] -= beta0 * gamma[0] * L
     return gamma
+
+
+@nb.njit(cache=True)
+def gamma_variation_qed(gamma, order, nf, L, alphaem_running):
+    """
+    Adjust the anomalous dimensions with the scale variations.
+
+    Parameters
+    ----------
+        gamma : numpy.ndarray
+            anomalous dimensions
+        order : tuple(int,int)
+            perturbation order
+        nf : int
+            number of active flavors
+        L : float
+            logarithmic ratio of factorization and renormalization scale
+
+    Returns
+    -------
+        gamma : numpy.ndarray
+            adjusted anomalous dimensions
+    """
+    if not alphaem_running:
+        # if alphaem is fixed then only alphas is varied so gamma[0,1] and gamma[0,2]
+        # don't get a variation while gamma[1,1] gets a variation that is O(as2aem1)
+        # that we are neglecting
+        gamma[1:, 0] = gamma_variation(gamma[1:, 0], order, nf, L)
+        return gamma
+    else:
+        beta0qcd = beta.beta_qcd((2, 0), nf)
+        beta1qcd = beta.beta_qcd((3, 0), nf)
+        beta2qcd = beta.beta_qcd((4, 0), nf)
+        beta0qed = beta.beta_qed((0, 2), nf)
+        # beta1qed = beta.beta_qcd((0, 3), nf)
+        # beta01qcd = beta.beta_qcd((2, 1), nf)
+        if order[0] >= 4:
+            gamma[4, 0] -= (
```

Comment:
here we are repeating the same formulas of the QCD only scale variation, can you avoid this?